src/BookNavigator/volumes/volumes-provider.js

```diff
@@ -24,40 +24,52 @@ export default class VolumesProvider {
     this.id = 'volumes';
     this.label = `Viewable files (${this.volumeCount})`;
     this.icon = html`${volumesIcon}`;
-    this.actionButton = this.headerIcon;
-    this.sortVolumes(true);
+
+    this.sortOrderBy = "";
```

Comment:
Add a type here and let's explicitly specify the default.

```suggestion
    /** @type {'initial' | 'asc' | 'desc'} */
    this.sortOrderBy = "";
```